packages/create-sitecore-jss/src/templates/nextjs-sxa/package.json

```diff
@@ -0,0 +1,15 @@
+﻿{
```

Comment:
I think we can get rid of this package.json example. It's referencing things that aren't used or don't exist.